tests/unit/test_sns_listener.py

```diff
@@ -86,3 +90,50 @@ def test_create_sns_message_body():
                                }
                            }})
     assert_equal(result, json.loads(expected))
+
+
+def test_create_sns_message_body_json_structure():
+    subscriber = {
```

Comment:
nitpick: exact same code is duplicated in lines 46-48, 96-100, 112-116, as well as 127-131 - perhaps we can create a small util function, something like `_get_sqs_subscriber_config()`?